Duplicati/Server/webroot/ngax/scripts/services/EditUriBuiltins.js

```diff
@@ -486,23 +513,48 @@ backupApp.service('EditUriBuiltins', function (AppService, AppUtils, SystemInfo,
             delete options[nukeopts[x]];
     }
 	
+	EditUriBackendConfig.parsers['storj'] = function (scope, module, server, port, path, options) {
+		if (options['--storj-auth-method'])
```

Comment:
Can we fix the indentation here?

Reply:
Fixed it. I don't understand why Notepad++ does this. I just copied the whole block and the first two lines have wrong indentation then.